geth/node/manager.go

```diff
@@ -563,3 +575,16 @@ func (m *NodeManager) RPCServer() (*rpc.Server, error) {
 
 	return m.rpcServer, nil
 }
+
+// Check if we have a node running and make sure is fully started
```

Comment:
The comment should have a format `isNodeAvailable checks if we have a running node and...`.